config/jobs/kubernetes-sigs/apiserver-network-proxy/apiserver-network-proxy-presubmits.yaml

```diff
@@ -16,3 +16,19 @@ presubmits:
       testgrid-dashboards: sig-cloud-provider-apiserver-network-proxy
       testgrid-tab-name: pr-test
       description: Tests the apiserver-network-proxy
+  - name: pull-apiserver-network-proxy-docker-build-all
+    always_run: true
+    skip_report: false
+    decorate: true
+    path_alias: sigs.k8s.io/apiserver-network-proxy
+    spec:
+      containers:
+      - image: golang:1.12
```

Comment:
This image should have no effect on docker builds right? We're using [1.13](https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/master/artifacts/images/agent-build.Dockerfile#L2), but I'm assuming the entire build will happen inside the go 1.13 docker container.

Reply:
Yep it should. We should bump the go version of the tests though 😅